r_basics_introduction/r_basics_introduction.md

```diff
@@ -0,0 +1,784 @@
+Importing Data is the first step of data analysis. Use `read_csv()` from the tidyverse package to import data stored in a CSV file.
+</div>
+</div>
+
+
+## Additional Resources
+
+<h3>Cheat Sheets</h3>
+
+RStudio's Data Import cheat sheet helps with importing data from file types similar to CSV but with other delimiters.  Cheat sheets for the RStudio IDE and Markdown can reinforce what you've learned so far in this module.
+
+To find official cheat sheets, go to the Help menu and choose "Cheat Sheets".  You can look around the short list provided.  You can also "Browse", and in the RStudio website you'll find other cheat sheets as well, including one for
+
+![RStudio help menu, with Cheat Sheets selected and the submenu option "Browse Cheat Sheets" selected](media/cheat_sheet_browse.png)<!-- style = "max-width:700px;" -->
+
+### More Data Types
+
+<h3>File Formats</h3>
+
+Not all data is in CSV files, and fortunately R supports a huge number of other file formats, and here are some of the packages that help import or export them:
+
+* Readxl: Microsoft Excel
+* Haven: SPSS, Stata, and SAS files
+* Googlesheets: Google Sheets
+* Rvest: web scraping
+* Jsonlite: JSON data and many APIs on the web
+
+![Hexagonal logos of the five packages listed above](media/file_types.png)<!-- style = "max-width:600px;" -->
+
+<h3>Databases</h3>
+
+You can also connect to a large number of **databases** directly in R to pull your data. For example, you can connect to SQL databases including MySQL and Oracle, as well as using APIs (application programming interfaces) for data collection applications like REDCap and websites with API access like PubMed or the New York Times.
+
+![Logos of many different database solutions](media/databases.png)
+
+### Additional Features
+
+<h3>Output Document Options</h3>
+
+<div style = "align-items: center; display: flex;">
+<div style = "margin: 1rem; max-width: 50%; float:left;">
+In addition to HTML, R Markdown documents can be "knitted" into a number of additional formats including PDF, Microsoft Word, PowerPoint, and even interactive dashboards.
+</div>
+<div style = "margin: 1rem; max-width: 50%; float:left;">
+![R Markdown logo surrounded by the logos of output document options, including HTML5, .pdf, and others.](media/document_output.png)<!-- style = "max-width: 400px;" -->
+</div>
+</div>
+
+<h3>Language Support</h3>
+
+Finally, in addition to R, R Markdown supports many other programming languages, including Python.
+
+So you can mix code chunks written in R with code chunks written in, say, Python:
+
+<div style = "align-items: center; display: flex;">
+
+<div style = "margin: 1rem; max-width: 50%; float:left;">
+![R logo with cartoon snake](media/r_python.png)<!-- style = "max-width: 400px;" -->
+</div>
+<div style = "margin: 1rem; max-width: 50%; float:left;">
+
+<lia-keep>
+<pre>
+```{python}
+import pandas
+covid_testing.info()
+```
+</pre>
+</lia-keep>
+</div>
+</div>
+
```

Comment:
most of this content does not belong in additional resources -- this should go in another section. 

Also do you have any other resources we can link to currently in the additional resources? (internally + externally)

Reply:
Agree, I've moved this section up, and slightly beefed up the "cheat sheets" section in Additional Resources.